edenai_apis/apis/openai/openai_video_api.py

```
@@ -0,0 +1,107 @@
+        file_url: Optional[str] = None,
+        model: Optional[str] = None,
+        **kwargs,
+    ) -> AsyncLaunchJobResponseType:
+        url = "https://api.openai.com/v1/videos"
+        payload = {
+            "prompt": text,
+            "model": model,
+            "seconds": str(duration),
+            "size": dimension,
+        }
+        response = requests.post(url=url, headers=self.headers, json=payload)
```

Ruff (0.13.3)

Probable use of `requests` call without timeout

(S113)